reference/5.1/Microsoft.PowerShell.Core/About/about_Arithmetic_Operators.md

```diff
@@ -62,12 +61,15 @@ The bitwise operators only work on integer types.
 
 PowerShell processes arithmetic operators in the following order:
 
-|Precedence|Operator       |Description                            |
-|----------|---------------|---------------------------------------|
-|1         | `()`          |Parentheses                            |
-|2         | `-`           |For a negative number or unary operator|
-|3         | `*`, `/`, `%` |For multiplication and division         |
-|4         | `+`, `-`      |For addition and subtraction           |
+|Precedence|Operator          |Description                            |
+|----------|------------------|---------------------------------------|
+|1         | `()`             |Parentheses                            |
+|2         | `-`              |For a negative number or unary operator|
+|3         | `*`, `/`, `%`    |For multiplication and division        |
+|4         | `+`, `-`         |For addition and subtraction           |
+|5         | `-band`, `-bnot` |For bitwise operations                 |
+|          | `-bor`, `-bxor`  |                                       |
+|          | `-shr`, `-shl`   |                                       |
```

Comment:
Since these render as rows, I'd add the 5 to each for context (instead of it looking like they have no precedence).  Or switch to `:::` column format so you can column row span that content.

Reply:
Can't use `:::` because it won't convert properly for `Get-Help`. Added text to each column